.goreleaser.yaml

```diff
@@ -27,30 +27,45 @@ builds:
     goarch:
       - amd64
       - arm64
-  
+
 partial:
   by: target
 
 archives:
   - id: defradb_playground
-    builds: 
+    builds:
       - defradb_playground
     format: binary
     # this name template makes the OS and Arch compatible with the results of `uname`.
-    name_template: '{{ .Binary }}_playground_{{ .Version }}_{{ .Os }}_{{- if eq .Arch "amd64" }}x86_64{{- else }}{{ .Arch }}{{ end }}{{- if .Arm }}v{{ .Arm }}{{ end }}'
+    name_template: >-
+      {{ .Binary }}_playground_{{ .Version }}_{{ .Os }}_
+      {{- if eq .Arch "amd64" }}x86_64
+      {{- else }}{{ .Arch }}{{ end }}
+      {{- if .Arm }}v{{ .Arm }}{{ end }}
   - id: defradb
-    builds: 
+    builds:
       - defradb
     format: binary
     # this name template makes the OS and Arch compatible with the results of `uname`.
-    name_template: '{{ .Binary }}_{{ .Version }}_{{ .Os }}_{{- if eq .Arch "amd64" }}x86_64{{- else }}{{ .Arch }}{{ end }}{{- if .Arm }}v{{ .Arm }}{{ end }}'
+    name_template: >-
+      {{ .Binary }}_{{ .Version }}_{{ .Os }}_
+      {{- if eq .Arch "amd64" }}x86_64
+      {{- else }}{{ .Arch }}{{ end }}
+      {{- if .Arm }}v{{ .Arm }}{{ end }}
 
 release:
   target_commitish: '{{ .Commit }}'
-  header: |
-    DefraDB v{{ .Major }}.{{ .Minor }} is a major pre-production release. Until the stable version 1.0 is reached, the SemVer minor patch number will denote notable releases, which will give the project freedom to experiment and explore potentially breaking changes.
+  header: >
+    DefraDB v{{ .Major }}.{{ .Minor }} is a major pre-production release.
```

Comment:
thought: This formatting looks weird with how we now have line return after ever comma and period.

Reply:
https://stackoverflow.com/a/21699210/18904980

Reply:
> Until the stable version 1.0 is reached, the SemVer minor patch number will denote notable releases,
which will give the project freedom to experiment and explore potentially breaking changes.

Looks nicer IMO.

Reply:
> > Until the stable version 1.0 is reached, the SemVer minor patch number will denote notable releases,
> > which will give the project freedom to experiment and explore potentially breaking changes.
> > To get a full outline of the changes, we invite you to review the official changelog below.
> > This release does include a Breaking Change to existing v{{ .Major }}.{{ .Minor }}.x databases.
> > If you need help migrating an existing deployment, reach out at [hello@source.network](mailto:hello@source.network) or join
> > our Discord at [discord.gg/w7jYQVJ](https://discord.gg/w7jYQVJ/).
> 
> Looks nicer IMO.

I have no hard preferences here haha, I have fixed it to this recommendation as looks nicer :) cheers